src/kernels/variables/debuggerVariables.ts

```diff
@@ -297,8 +297,7 @@ export class DebuggerVariables
         this.importedGetVariableInfoScriptsIntoKernel.delete(key);
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    private async evaluate(code: string, frameId?: number): Promise<any> {
+    async evaluate(code: string, frameId?: number): Promise<DebugProtocol.EvaluateResponse['body']> {
```

Comment:
I'm exposing this function so I can use it in the debugger path within the data viewer dependency service.

Reply:
Not a fan of this